deploy/olm-catalog/integreatly-operator/integreatly-operator-1.13.0/integreatly-operator.v1.13.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,461 @@
+          - list
+          - watch
+          - update
+          - create
+        - apiGroups:
+          - build.openshift.io
+          resources:
+          - jenkins
+          verbs:
+          - '*'
+        serviceAccountName: integreatly-operator
+      deployments:
+      - name: integreatly-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: integreatly-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: integreatly-operator
+            spec:
+              containers:
+              - command:
+                - integreatly-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: integreatly-operator
+                image: quay.io/integreatly/integreatly-operator:v1.13.0
+                imagePullPolicy: Always
+                name: integreatly-operator
+                resources: {}
+              serviceAccountName: integreatly-operator
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - '*'
+        - apiGroups:
+          - ""
+          resources:
+          - namespaces
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - apps
+          resourceNames:
+          - integreatly-operator
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - integreatly.org
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: integreatly-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: true
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - integreatly
+  labels:
+    alm-owner-integreatly: integreatly-operator
+    operated-by: integreatly-operator
+  maintainers:
+  - email: integreatly-support@redhat.com
+    name: Integr8ly
+  maturity: alpha
+  minKubeVersion: 1.11.0
```

Comment:
@philbrookes Sorry, missed this on the first pass. This is no longer the case with the new CRD. min version is 1.12.0

Reply:
nice catch, fixed now.